packages/frontend/app/components/terminal.tsx

```diff
@@ -23,7 +24,11 @@ export function Terminal({ children }: TerminalProps) {
         <div className="flex justify-between items-center bg-gray-200 dark:bg-green-900 p-2 rounded-t-lg">
           <div className="flex items-center space-x-4">
             <div className="flex space-x-2">
-              <div className="w-3 h-3 rounded-full bg-red-500"></div>
+              <div
```

Comment:
maybe a button for focus / accessibility?

Reply:
yeah let's just do a semantics/a11y pass at the end